irohad/torii/processor/impl/query_processor_impl.cpp

```diff
@@ -35,13 +36,30 @@ namespace iroha {
                   shared_model::interface::StatefulFailedErrorResponse>()
               .build());
     }
+    std::shared_ptr<shared_model::interface::BlockQueryResponse>
+    buildBlocksQueryError(const std::string &message) {
+      return clone(shared_model::proto::BlockQueryResponseBuilder()
+                       .errorResponse(message)
+                       .build());
+    }
+    std::shared_ptr<shared_model::interface::BlockQueryResponse>
+    buildBlocksQueryBlock(shared_model::interface::Block &block) {
+      return clone(shared_model::proto::BlockQueryResponseBuilder()
+                       .blockResponse(block)
+                       .build());
+    }
```

Comment:
Looks like code duplication (see above)
Is there a better way to do that?

Reply:
Any suggestions?